packages/shared-core/src/filters.ts

```diff
@@ -174,10 +174,10 @@ export const buildLuceneQuery = (filter: SearchFilter[]) => {
           return
         }
       }
-      if (type === "number" && typeof value === "string") {
+      if (type === "number" && typeof value === "string" && !isHbs) {
```

Comment:
this feels like it could do with a test - I don't really track what this logic is trying to achieve and why the `!isHbs` needed to be removed - its likely someone else could be confused by this - a quick unit test could (we already have a filters test set I think) could help avoid a regression here.

Reply:
Unit tests added